src/lib/crossPlatformFetch.ts

```diff
@@ -25,10 +24,6 @@ export default async function crossPlatformFetch<T>(
 
     response = await isomorphicFetch(url, opts)
   } else {
-    // Add credentials: 'include' to options
-    // We send the token with the headers only in development mode, DO NOT
-    // DO THIS IN PRODUCTION, as it is a security risk.
-    opts.headers = headers(env().DEV ? token : undefined)
```

Comment:
I'm not sure why we ever needed to do this.

Reply:
Hmm I am gonna have to look at this, I am confused what this is doing.

Reply:
I have brought back these headers after our [Slack](https://kittycadworkspace.slack.com/archives/C07A80B83FS/p1752248033631379?thread_ts=1752186947.963619&cid=C07A80B83FS) discussion.